ospf6d/ospf6_flood.c

```diff
@@ -348,10 +348,10 @@ void ospf6_flood_interface(struct ospf6_neighbor *from, struct ospf6_lsa *lsa,
 					if (req == on->last_ls_req) {
 						ospf6_lsa_unlock(req);
 						on->last_ls_req = NULL;
-					}
-					if (req)
+					} else {
```

Comment:
Why is this converted to an else?  Shouldn't the if statement just be removed and we always just call ospf6_lsdp_remove?

Reply:
You're right, thank you. I'll add a req = NULL after the ospf6_lsa_unlock()